src/main/java/com/spotinst/sdkjava/model/OceanK8sVirtualNodeGroupService.java

```diff
@@ -0,0 +1,166 @@
+package com.spotinst.sdkjava.model;
+
+import com.spotinst.sdkjava.client.response.BaseServiceEmptyResponse;
+import com.spotinst.sdkjava.client.response.BaseSpotinstService;
+import com.spotinst.sdkjava.client.rest.*;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.api.ocean.kubernetes.ApiK8sVirtualNodeGroup;
+import com.spotinst.sdkjava.model.responses.OceanK8sVirtualNodeGroupApiResponse;
+import org.apache.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class OceanK8sVirtualNodeGroupService extends BaseSpotinstService {
```

Comment:
List method also need to be implemented.

Reply:
List method has added